### PR TITLE
don't exit on errors or empty sqs receives

### PR DESCRIPTION
**Pull request by david-fritz-gravwell**

### CODE REVIEW

**Comment by david-fritz-gravwell** [comment]
@kris-watts-gravwell PTAL

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

File: ingesters/sqsIngester/main.go [hunk 1]
```diff
@@ -196,7 +197,9 @@ func queueRunner(hcfg *handlerConfig) {
 		select {
 		case out = <-c:
 			if out == nil {
-				return
+				lg.Error("received empty SQS response")
+				sleepContext(cts, ERROR_BACKOFF)
```
Comment (kris-watts-gravwell) [inline ingesters/sqsIngester/main.go:201]: `ctx` not `cts`

**Review by kris-watts-gravwell — CHANGES_REQUESTED** [review]
typo

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

**Pull request merged**